pysal/contrib/geotable/utils.py

```diff
@@ -24,15 +24,14 @@ def to_df(df, geom_col='geometry', **kw):
     """
     import pandas as pd
     from geopandas import GeoDataFrame, GeoSeries
-    df[geom_col] = df[geom_col].apply(pShape)
-    if isinstance(df, (GeoDataFrame, GeoSeries)):
-        df = pd.DataFrame(df, **kw)
-    return df
+    out = df.copy()
```

Comment:
In order to shield against potential future API changes, should we explicitly make it a deep copy?

`out = df.copy(deep=True)`

`pandas` relevant doc [here](http://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.copy.html)